### DISCUSSION TITLE
"Try it Now" on README.md results in error.

### DISCUSSION THREAD

**Discussion by ghost** [idom-team/idom#960] [Problem] [answered]
Follow the link:
https://mybinder.org/v2/gh/reactive-python/reactpy-jupyter/main?urlpath=lab/tree/notebooks/introduction.ipynb

and you will receive the following error:

```
Could not resolve ref for gh:reactive-python/reactpy-jupyter/main. Double check your URL. GitHub recently changed default branches from "master" to "main". Tip: HEAD will always resolve to a repository's default branch.
Trying again with HEAD instead of main. Please update your links.
Could not resolve ref for gh:reactive-python/reactpy-jupyter/main. Double check your URL. GitHub recently changed default branches from "master" to "main". Tip: HEAD will always resolve to a repository's default branch.
```

**Comment by rmorshea** [comment]
Apologies. We're working on renaming this project from IDOM so some links may be broken temporarily. This link will take you to a Jupyter Notebook that uses the old `idom` name:

https://mybinder.org/v2/gh/idom-team/idom-jupyter/main?filepath=notebooks%2Fintroduction.ipynb

**Reply by rmorshea** [reply, marked as answer]
This has now been fixed. New link: https://mybinder.org/v2/gh/reactive-python/reactpy-jupyter/main?urlpath=lab/tree/notebooks/introduction.ipynb
